src/main/java/io/gatling/mojo/VerifyMojo.java

```diff
@@ -63,7 +63,8 @@ private void verifyLastRun() throws IOException, MojoFailureException, MojoExecu
 
     private void checkError(String line) throws MojoFailureException {
         if (StringUtils.contains(line, LAST_RUN_FILE_ERROR_LINE)) {
-            throwFailureException("Gatling had errors during execution!");
+            throwFailureException(line.
+                    replaceFirst(LAST_RUN_FILE_ERROR_LINE, ""));
```

Comment:
line.substring(LAST_RUN_FILE_ERROR_LINE.length())

Reply:
Yep, much better, fixed in new commit